topics/perl6/index.md

```diff
@@ -0,0 +1,14 @@
+---
+aliases: rakudo, nqp
+created_by: Larry Wall
+display_name: Perl 6
+github_url: https://github.com/perl6
+logo: perl6.png
+related: language, perl
+released: December 25, 2015
+short_description: Perl 6 is an expressive and feature-rich programming language.
+topic: perl6
+url: https://www.perl6.org/
+wikipedia_url: https://en.wikipedia.org/wiki/Perl_6
+---
+Perl 6 is an expressive and feature-rich programming language designed by Larry Wall and developed by the Perl 6 community. Features include: Object-oriented programming, functional programming primitives, parallelism, concurrency, asynchrony, definable grammars for pattern matching and generalized string processing, optional and gradual typing.
```

Comment:
Could you update this to:

- Lowercase "Object" in "include: Object-oriented"
- Add "and" before "optional and gradual typing"

Reply:
@cheshire137 Done. Thanks a lot.